stories/assets/js/footer_custom.js

```diff
@@ -0,0 +1,15 @@
+/* footer menu on mobile view start */
+export function FooterNav() {
+  $(window).on('load resize', () => {
+    if ($(window).width() < 768) {
+      $('.footer__items li > h6').click(function () {
+        $('li').removeClass('active').find('.footer__panel').stop()
+          .slideUp();
+        $(this).closest('li').toggleClass('active').find('.footer__panel')
+          .stop()
+          .slideToggle();
+      });
+    }
+  });
+}
```

Comment:
@Shubham-Sharma-77 Code is quite similar to the faq code so can we make these codes generic for accordion and use only single code at both places

Reply:
@surbhi90 ok but how we can run the similar js on mobile screen on footer navigation.